miden-lib/asm/miden/kernels/tx/prologue.masm

```diff
@@ -605,7 +605,7 @@ proc.process_input_note
     # => [NULLIFIER, note_ptr, idx]
 
     # save nullifier to memory
-    movup.6 exec.memory::get_consumed_note_nullifier_ptr
```

Comment:
This was using the value duplicated here:

https://github.com/0xPolygonMiden/miden-base/blob/4e982bfc6480db98233719e0e4c02bd729e9ee0e/miden-lib/asm/miden/kernels/tx/prologue.masm#L797